projects/observability/src/pages/apis/topology/application-flow.component.ts

```diff
@@ -15,203 +19,221 @@ import {
   template: `
     <div class="vertical-flex-layout">
       <ht-page-header></ht-page-header>
-      <ht-navigable-dashboard [navLocation]="this.location" [defaultJson]="this.defaultJson"> </ht-navigable-dashboard>
+      <ht-navigable-dashboard
+        *htLetAsync="this.getDefaultJson() as defaultJson"
+        [navLocation]="this.location"
+        [defaultJson]="this.getDefaultJson()"
```

Comment:
Please do this
```
[defaultJson]="this.getDefaultJson | htMemoize | async
```

Reply:
can't use htMemoize here as we are using this.isGraphLayoutEnabled() here

Reply:
should I bring that inside the function? I kept it outside to keep it readable